Sources/AsyncHTTPClient/HTTPClient.swift

```diff
@@ -561,12 +569,6 @@ public class HTTPClient {
             }
 
             return future.flatMap {
-                let taskHandler = TaskHandler(task: task,
```

Comment:
was this move from inside the closure to outside the closure deliberate?

Reply:
yes, we need task handler to access `calloutTo` methods